### DISCUSSION TITLE
Should the System DB and Tables Use ReplicatedMergeTree when Scaling Out?

### DISCUSSION THREAD

**Discussion by tomtom215** [ClickHouse/ClickHouse#62291] [Q&A] [answered]
Hello,

In the scenario that a single instance was deployed on K8's but then ClickHouse Keeper is enabled and replicas are created (but still single shard), the system DB & tables show they are using the MergeTree after the replicas were created and restarted.

Is it required to ALTER system tables in the SYSTEM DB to use ReplicatedMergeTree instead? We have not been able to find and documentation around ClickHouse-Keeper to show that there are steps required after enabling replicas on an existing deployment and I feel the SYSTEM DB is not being replicated.

**Comment by den-crane** [comment, marked as answer]
No. System database's tables should not be replicated.

If you want to be able to query data from all nodes for system tables you can use something like this https://kb.altinity.com/altinity-kb-setup-and-maintenance/sysall/

**Reply by tomtom215** [reply]
Wow, thanks for this documentation @den-crane ! 

We have tried it out successfully in our staging environment and plan to test in production this week. I will mark this as "answered" as you provided the direct answer to my question, but do you have any other recommendations like this that are necessary or best practice after going from a single instance to multiple replicas? 

For example: we know users, DB's and tables that were created before enabling replication need to be adjusted for "on cluster" use and using replicated merge trees, but is there anything else that should be considered or altered?
